src/components/AddPlaidBankAccount.js

```diff
@@ -12,66 +12,28 @@ import PlaidLink from './PlaidLink';
 import * as BankAccounts from '../libs/actions/BankAccounts';
 import ONYXKEYS from '../ONYXKEYS';
 import styles from '../styles/styles';
-import canFocusInputOnScreenFocus from '../libs/canFocusInputOnScreenFocus';
 import themeColors from '../styles/themes/default';
 import compose from '../libs/compose';
 import withLocalize, {withLocalizePropTypes} from './withLocalize';
 import Picker from './Picker';
+import plaidDataPropTypes from '../pages/ReimbursementAccount/plaidDataPropTypes';
 import Text from './Text';
-import * as ReimbursementAccountUtils from '../libs/ReimbursementAccountUtils';
-import ReimbursementAccountForm from '../pages/ReimbursementAccount/ReimbursementAccountForm';
 import getBankIcon from './Icon/BankIcons';
 import Icon from './Icon';
-import TextInput from './TextInput';
+import FullPageOfflineBlockingView from './FullPageOfflineBlockingView';
```

Comment:
```suggestion
import FullPageOfflineBlockingView from './BlockingViews/FullPageOfflineBlockingView';
```
FYI, was testing https://github.com/Expensify/Web-Expensify/pull/34260 on this branch locally and had to make this change to resolve console error